spec/unit/scrum/sprint_cleaner_spec.rb

```diff
@@ -15,7 +15,8 @@
 
     it 'moves remaining cards to target board', vcr: 'sprint_cleanup', vcr_record: false do
       expect(STDOUT).to receive(:puts).exactly(13).times
-      expect(subject.cleanup('7Zar7bNm', '72tOJsGS', set_last_sprint_label: true)).to be
+      set_last_sprint_label = true
+      expect(subject.cleanup('7Zar7bNm', '72tOJsGS', set_last_sprint_label)).to be
```

Comment:
what about 

```ruby
expect(subject.cleanup('7Zar7bNm', '72tOJsGS', true).to be
```
instead of an extra variable? :thinking:

Reply:
Actually, cornelius had earlier suggested that the extra verbosity, because he thought that the purpose of that third parameter wasn't evident. So he suggested moving to a named parameter.

Reply:
@Ana06 the expection arose due to using named parameters. when no value for the last parameter was passed it threw `ArgumentError`. Now that've moved to default params, that error won't be occuring anymore. I'm not sure how I'm supposed to write a test for that. :thinking:

Reply:
@ankitkataria The extra verbosity I suggested was not for the test. With a positional parameter you indeed should just call it without an extra variable. My point was about the signature of the function so that when you call it you are forced to provide a named parameter instead of a boolean. When you read the code where the function is called it's then obvious what it does. When there is only a `false` or `true` as a parameter you need to look up the definition of the function to understand what the calling code does. That's why for booleans named parameters are generally preferrable.

Regarding the test, what would be ideal is, if you write a test which fails with the old version of the code. That would expose the bug that the code didn't work anymore when no parameter was given. This test should go into the tests of the `Cli` class because that's where the issue occurred. When you then apply a fix the test will be green and it will be guaranteed that it won't occur again, because there is a test now. That's he gist of test-driven development.

As for the fix I think you should fix it in the `Cli` class when calling the method, not going back to positional parameters.

Hope that clear up what I was thinking :-)